utils/styleguide/setup.js

```diff
@@ -22,8 +22,8 @@ global.State = State;
 global.Grid = Grid;
 global.Row = styled(Row).attrs({
   alignItems: 'center'
-})``;
+});
```

Comment:
Does this break the `Rows` used in any of the examples? I thought you had to always end a styled-component with that template string to get a valid component.

Reply:
Ack. You're right. I'll take another crack at solving the linting error.